newrelic/common/package_version_utils.py

```diff
@@ -86,6 +84,15 @@ def _get_package_version(name):
                 if callable(version):
                     continue
 
+                # Some frameworks (such as `pypdfium2`) may use a class
+                # property to define the version.  Because class properties
+                # are not callable we need to check if the result is
+                # anything other than a string, tuple, or list.  If so,
+                # we need to skip this method of version retrieval and use
+                # `pkg_resources` or `importlib.metadata`.
+                if version and not isinstance(version, (str, tuple, list)):
+                    break
```

Comment:
```suggestion
                # Some frameworks (such as `pypdfium2`) may use a class
                # property to define the version.  Because class properties
                # are not callable we need to check if the result is
                # anything other than a string, tuple, or list.  If so,
                # we need to skip this method of version retrieval and use
                # `pkg_resources` or `importlib.metadata`.
                if version and not isinstance(version, (str, tuple, list)):
                    continue
```

Instead of `break` can we use `continue` to check if any of the other attributes are potentially the version we're looking for? This would let us combine these 2 logic blocks.

For example, if `module.version` is a module or some other object but also exports `module.version_tuple` which is just the tuple we're looking for.